test/jdk/javax/swing/Popup/TaskbarPositionTest.java

```diff
@@ -93,7 +118,8 @@ public TaskbarPositionTest() {
             screenBounds.y += screenInsets.top;
         }
 
-        setVisible(true);
+        frame.setLocationRelativeTo(null);
```

Comment:
This line puts the frame in the centre of the screen and the test becomes useless. The frame *must be located* at the bottom of the screen to verify the popup windows are shown on top of the taskbar / dock. In fact, the popups are located so that bottom of the popup is at the bottom of the combobox / menu item.